observation_portal/requestgroups/serializers.py

```diff
@@ -160,6 +166,26 @@ def validate(self, data):
             raise serializers.ValidationError(_(
                 'Currently only square binnings are supported. Please submit with bin_x == bin_y'
             ))
+
+        extra_param_max_exp_time = 0
+        for field, value in data.get('extra_params', {}).items():
+            try:
+                if 'exposure_time' in field and float(value) > extra_param_max_exp_time:
+                    extra_param_max_exp_time = float(value)
+            except ValueError:
+                pass
+        if extra_param_max_exp_time > 0:
+            data['exposure_time'] = extra_param_max_exp_time
+
+        return data
+
+    def to_internal_value(self, data):
+        data = super().to_internal_value(data)
+        for field, value in data.get('extra_params', {}).items():
+            try:
+                data['extra_params'][field] = float(value)
```

Comment:
This is so that extra params that are numbers are stored as such in the database json field (and then are retrieved that way as well)

Reply:
Sounds reasonable to add to other serializers.